stockflux-securities-master/src/components/ToggleSwitch.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./ToggleSwitch.css";
+import PropTypes from "prop-types";
+
+const ToggleSwitch = ({ onChange, id, value, className, ...props }) => {
+  return (
+    <>
+      <input
+        {...props}
+        value={value}
+        id={id}
+        type="checkbox"
+        className="input-form-toggle"
+        onChange={onChange}
+      />
+      <label className={`toggle-switch ${className}`} htmlFor={id} />
+    </>
+  );
+};
+
+ToggleSwitch.propTypes = {
+  onChange: PropTypes.func.isRequired,
+  id: PropTypes.string.isRequired,
+  value: PropTypes.bool.isRequired,
```

Comment:
Value shouldn't have to be a boolean type. Checkbox input fields are often given string values.

Reply:
I've made id required as without it the component doesn't function. I've made value and onChange optional though